composer.json

```diff
@@ -21,13 +21,12 @@
     ],
     "require": {
         "php": "^7.4",
-
         "friendsofsymfony/oauth-server-bundle": ">2.0.0-alpha.0 ^2.0@dev",
         "sensiolabs/security-checker": " ^5.0",
         "sylius/sylius": "dev-master",
         "symfony/flex": "^1.7",
-        "sylius/invoicing-plugin": "~0.10",
-        "sylius/refund-plugin": "^1.0.0@rc",
+        "sylius/invoicing-plugin": "~0.13",
+        "sylius/refund-plugin": "1.0.0-RC.7",
```

Comment:
I replaced `^1.0.0@rc` to `1.0.0-RC.7` because composer has the problem:
<img width="1440" alt="Screenshot 2021-03-04 at 11 29 16" src="https://user-images.githubusercontent.com/39232096/109950628-047f4b80-7cdd-11eb-91f1-b01de364802f.png">